config/jobs/kubeflow/kubeflow-periodics.yaml

```diff
@@ -182,6 +182,36 @@ periodics:
   annotations:
     testgrid-dashboards: sig-big-data
     description: Periodic testing of Kubeflow examples
+  #******************************************************************************
+  # kubeflow/kfp-functional-test
+  #******************************************************************************
+- name: kubeflow-pipelines-periodic-functional-test
+  cluster: kubeflow
+  interval: 10m
+  decorate: true
+  labels:
+    preset-service-account: "true"
+  extra_refs:
+    - org: kubeflow
+      repo: pipelines
+      base_ref: master
+      workdir: true
+  spec:
+    containers:
+      - image: gcr.io/ml-pipeline-test/kfp-functional-test-base:latest
+        imagePullPolicy: Always
+        command:
+          - "./test/kfp-functional-test/kfp-functional-test.sh"
+        env:
+          - name: REPO_OWNER
+            value: kubeflow
+          - name: REPO_NAME
```

Comment:
I followed this job https://github.com/kubernetes/test-infra/blob/1a136543a417284131f98ad3aad5481052be0ee2/config/jobs/kubeflow/kubeflow-periodics.yaml#L207 to set the REPO_OWNER and REPO_NAME, not sure whether I should change the REPO_NAME to "pipelines"